packages/react/src/components/MenuButton/_menu-button.scss

```diff
@@ -17,7 +20,7 @@
 
     .#{$prefix}--btn--ghost.#{$prefix}--btn--icon-only {
       background-color: $ui-01;
-      @include box-shadow;
+      @include box-shadow();
```

Comment:
In another file we removed the parens and here we added. Both did not have any arguments. Is this on purpose?

Reply:
This is the result of a merge, and actually should have thrown a stylelint error, because new recommendations are to remove them if no params are passed... 🤔 I wonder if we have a disable somewhere that didn't get re-enabled. I'll check on that.